examples/src/main/scala/org/apache/spark/examples/SparkKMeans.scala

```diff
@@ -82,7 +82,7 @@ object SparkKMeans {
     while(tempDist > convergeDist) {
       val closest = data.map (p => (closestPoint(p, kPoints), (p, 1)))
 
-      val pointStats = closest.reduceByKey{case ((p1, c1), (p2, c2)) => (p1 + p2, c1 + c2)}
+      val pointStats = closest.reduceByKey(mergeResults)
```

Comment:
Not quite sure why, but a few calls to `reduceByKey` didn't like the existing syntax in 2.13. I had to break out a typed method. `missing parameter type for expanded function`